src/test/java/uk/gov/justice/digital/delius/OffendersResource_GetLatestRecallAndReleaseForOffender.java

```diff
@@ -234,6 +242,44 @@ public void getLatestRecallAndReleaseForOffender_notSingleCustodyEvent_returnsDe
 
     }
 
+    @Test
+    public void getLatestRecallAndReleaseForOffender_custodyRecordNotFound_returnsBadRequest() {
+        org.mockito.BDDMockito.given(mockOffenderService.offenderIdOfCrn(anyString()))
+                .willReturn(MAYBE_ANY_OFFENDER_ID);
+        org.mockito.BDDMockito.given(mockOffenderService.getOffenderLatestRecall(ANY_OFFENDER_ID))
+                .willThrow(CustodyService.CustodyNotFoundException.class);
+        given()
+                .auth()
+                .oauth2(validOauthToken)
+                .contentType(APPLICATION_JSON_VALUE)
+                .when()
+                .get("/offenders/crn/X320741/release")
+                .then()
+                .statusCode(400);
+    }
+
+    @Test
+    public void getLatestRecallAndReleaseForOffender_custodyRecordNotFound_returnsEventIdInErrorMessage() {
+        org.mockito.BDDMockito.given(mockOffenderService.offenderIdOfCrn(anyString()))
```

Comment:
any reason for fully qualifying `BDDMockito` here?

Reply:
Just because we're using RestAssured's given to test the GET call.  Unless there's a better way I'm missing?